features/ui-common/src/main/kotlin/com/wire/android/ui/theme/Theme.kt

```diff
@@ -18,6 +18,7 @@
 
 package com.wire.android.ui.theme
 
+//import com.wire.android.navigation.rememberNavigator
```

Comment:
🤔

Reply:
I commented it for now, since it was only used for `Previews` as you can see down below in line ~50.
We need at some point to bring the navigation as well to a separate module, but for now I wanted to keep it simple and less invasive 😅  hope is not a problem for the team. (actually removing it, does not break previews).